zk-tests/src/Cheatcodes.t.sol

```diff
@@ -241,4 +241,26 @@ contract ZkCheatcodesTest is Test {
             address(0x4e59b44847b379578588920cA78FbF26c0B4956C).balance
         );
     }
+
+    function testRecordLogsInZkVm() public {
+        // ensure we are in zkvm
+        (bool _success, bytes memory _ret) = address(vm).call(abi.encodeWithSignature("zkVm(bool)", true));
```

Comment:
Thanks for adding this, but I'm not sure if we should maintain zk-tests now if `cargo test` works. I plan to remove zk-tests soon once everything is migrated.

Reply:
For consistency, and we can just remove everything at once